src/Servers/Kestrel/test/FunctionalTests/RequestTests.cs

```diff
@@ -500,6 +500,7 @@ public async Task AbortingTheConnectionSendsFIN()
 
         [Theory]
         [MemberData(nameof(ConnectionMiddlewareData))]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/23043")]
         public async Task ConnectionClosedTokenFiresOnClientFIN(ListenOptions listenOptions)
```

Comment:
This was unquarntined less than 24 hours ago because it hadn't failed in 30+ days. Given that it's a little surprising it failed again so quickly. Do we think this is just because quarantined tests get run less? Or could there be something environmental going on that doesn't happen in our quarantined test pipeline?

Reply:
Hmm.  They actually run at about the same frequency in the quarantine runs, so it's possible there is something environmental :(  Do the logs in the linked issue give any hints?

Reply:
This one is error on my part. This test has only been passing since 6/2/2020. I found [this issue](https://github.com/dotnet/aspnetcore/issues/20163). It looks like this test was first quarantined in March, unquarantined in May, then quarantined again in June.

Reply:
I'm also curious why it was removed from quarantine at all. It was put into quarantine on 6/1, so it hasn't passed for 30 days in a row.